public/app/plugins/panel/annolist/plugin.json

```diff
@@ -2,6 +2,7 @@
   "type": "panel",
   "name": "Annotations list",
   "id": "annolist",
+  "alias": "ryantxu-annolist-panel",
```

Comment:
shouldn't this be:

```
  "id": "grafana-annolist-panel",
  "alias": "annolist",
```
?

Reply:
we could do that -- but that is a different project :)   The goal here is not to rename the ids, just setting alias in plugin.json rather than hardcoded